lib/fast_gettext/translation.rb

```diff
@@ -72,6 +77,14 @@ def ns_(*args, &block)
       # block is called once again to compare result
       block && translation == block.call ? translation : translation.split(NAMESPACE_SEPARATOR).last
     end
+    alias :nsgettext :ns_
+
+    def np_(*args, &block)
+      context, key = args[0..1]
+      nargs = ["#{context}#{CONTEXT_SEPARATOR}#{key}"] + args[2..-1]
+      n_(*nargs, &block) or (block ? block.call : key)
```

Comment:
n_ already handles the block, so don't call it ?

Reply:
will fix